skfda/preprocessing/registration/_landmark_registration.py

```diff
@@ -7,6 +7,7 @@
 
 from ... import FDataGrid
 from ...representation.interpolation import SplineInterpolator
+from ...utils import constants
```

Comment:
I would rather put `constants` inside the `misc` module.

Reply:
Neither `constants.py` nor `_utils.py` can be moved to `misc` due to dependencies. I created a internal package `_utils` containing `constants.py` and `_utils.py` with utilities used by developers.